swapi/.eslintrc.json

```diff
@@ -0,0 +1,48 @@
+{
+  "root": true,
+  "ignorePatterns": [
+    "projects/**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts"
+      ],
+      "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@angular-eslint/recommended",
+        "plugin:@angular-eslint/template/process-inline-templates",
+        "prettier"
```

Comment:
i guess prettier alone is redundant , please try eslint-prettier plugin , in order to make all code quality stuff via eslint